tests/data/test31

```diff
@@ -100,36 +100,18 @@ Accept: */*
 # https://curl.haxx.se/docs/http-cookies.html
 # This file was generated by libcurl! Edit at your own risk.
 
-127.0.0.1	FALSE	/we/want/	TRUE	0	securewithspace	after
 127.0.0.1	FALSE	/we/want/	FALSE	0	prespace	yes before
 127.0.0.1	FALSE	/we/want/	FALSE	0	withspaces2	before equals
 127.0.0.1	FALSE	/we/want/	FALSE	0	withspaces	yes  within and around
 127.0.0.1	FALSE	/we/want/	FALSE	0	blexp	yesyes
 #HttpOnly_127.0.0.1	FALSE	/silly/	FALSE	0	magic	yessir
 127.0.0.1	FALSE	/we/want/	FALSE	2054030187	nodomain	value
 127.0.0.1	FALSE	/	FALSE	0	partmatch	present
-#HttpOnly_127.0.0.1	FALSE	/p4/	TRUE	0	httpandsec8	myvalue9
-#HttpOnly_127.0.0.1	FALSE	/p4/	TRUE	0	httpandsec7	myvalue8
-#HttpOnly_127.0.0.1	FALSE	/p4/	TRUE	0	httpandsec6	myvalue7
-#HttpOnly_127.0.0.1	FALSE	/p4/	TRUE	0	httpandsec5	myvalue6
-#HttpOnly_127.0.0.1	FALSE	/p4/	TRUE	0	httpandsec4	myvalue5
-#HttpOnly_127.0.0.1	FALSE	/p4/	TRUE	0	httpandsec3	myvalue4
-#HttpOnly_127.0.0.1	FALSE	/p4/	TRUE	0	httpandsec2	myvalue3
-#HttpOnly_127.0.0.1	FALSE	/p4/	TRUE	0	httpandsec	myvalue2
```

Comment:
The reason for these large sections removed is that they are setting secure cookies over HTTP, which with this patch is no longer allowed.